content/Solutions/Integrations/Veeam.md

```diff
@@ -107,7 +107,7 @@ Recommended sizing:
   The number of VMs is based upon conservative throughput estimates with an average VM size set as 100GB and a backup window of eight hours running full backups.
   All other requirements for the number of Veeam backup proxies, and networking dependencies also apply.
 
```

Comment:
in the bullet point in line 105, as a sentence with a period don't capitalize only
```suggestion
* Backup only? Assumes using the storage only as a backup repository.
```